apps/site/lib/site_web/controllers/page_controller.ex

```diff
@@ -36,6 +36,7 @@ defmodule SiteWeb.PageController do
     |> async_assign_default(:promoted_items, fn -> promoted end)
     |> async_assign_default(:whats_happening_items, fn -> remainder end)
     |> async_assign_default(:alerts, fn -> Alerts.Repo.all(conn.assigns.date_time) end)
+    |> async_assign_default(:event_teasers, fn -> CMS.Repo.next_n_event_teasers(3) end, [])
```

Comment:
The acceptance criteria in the ticket mentions showing 6 events on larger screens. We should probably bump this to 6 and then reduce the events visible on smaller screens